home/src/main/resources/rdf/i18n/en_US/interface-i18n/firsttime/vitro_UiLabel.ttl

```diff
@@ -254,6 +254,102 @@ uil-data:reset_password_note.Vitro
         uil:hasKey      "reset_password_note" ;
         uil:hasPackage  "Vitro-languages" .
 
+uil-data:password_reset_email_sent.Vitro
+        rdf:type         owl:NamedIndividual ;
+        rdf:type         uil:UILabel ;
+        rdfs:label       "Password recovery email is sent to "@en-US ;
```

Comment:
"Password recovery email _has been_ sent to " or "A password recovery email has been sent to "

Reply:
Done